api/swagger/swagger.yaml

```diff
@@ -294,6 +294,23 @@ paths:
           schema:
             $ref: "#/definitions/ErrorResponse"
 
+  /sources:
+    x-swagger-router-controller: sources
+    get:
+      tags:
+        - sources
+      description: Returns list of sources
+      operationId: list
```

Comment:
I also like the idea of being semantically nice to write / read as in your example. I just couldn't figure out a way to tackle both issues, and adhering to the Swagger specifications was just higher priority when I operated those.